<launch>
    <arg name="DEBUG_OUTPUT" default="false"/>
    <arg name="config" default="$(find aurora_sensor)/config/config.yaml"/>
    <arg name="base_link_name" default="ndi_base_link"/>
    <arg name="use_rivz" default="false"/>
    
    <node name="aurora_sensor" pkg="aurora_sensor" type="broadcast_aurora.py" output="screen">
        <rosparam file="$(arg config)" command="load" />
        <param name="base_link_name" value="$(arg base_link_name)"/>
        <param name="DEBUG_OUTPUT" value="$(arg DEBUG_OUTPUT)"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find aurora_sensor)/config/sample.rviz" if="$(arg use_rivz)" /> 

</launch>